models/evc.py

```diff
@@ -171,6 +170,36 @@ def sync(self, keys: set = None):
             return
         self._mongo_controller.upsert_evc(self.as_dict())
 
+    def _get_unis_use_tags(self, **kwargs) -> (UNI, UNI):
+        """Obtain both UNIs (uni_a, uni_z).
+        If a UNI is changing, verify tags"""
+        uni_a = kwargs.get("uni_a", None)
+        uni_a_flag = False
+        if uni_a and uni_a != self.uni_a:
+            uni_a_flag = True
+            try:
+                self._use_uni_vlan(uni_a)
```

Comment:
Looks good. In the future when we start supporting vlan range, we'll need to try to use only the difference of the new - existing, we'll get there in the future, good thing that it's structured well.

Reply:
A'right